src/qz/printer/status/printer/CupsPrinterStatusMap.java

```diff
@@ -57,17 +859,18 @@
         this.parent = parent;
     }
 
-    public static NativePrinterStatus match(String code) {
+    public static NativeStatus match(String code, String state) {
```

Comment:
What's the purpose of changing the return type to the parent type?

Reply:
It brings it into line with the other match functions